docs/kb/compaction.rst

```diff
@@ -188,7 +188,31 @@ Which Strategy is best to use
 -----------------------------
 Use the table in :ref:`Which strategy is best <which-strategy-is-best>` to determine the right strategy for your needs.
 
+.. _compaction-off-strategy-compaction:
 
+Off-strategy Compaction
+----------------------------
+
+Off-strategy compaction is an internal mechanism used by ScyllaDB to improve the efficiency of node operations, 
+such as ``bootstrap``, ``repair``, or ``replace``. It keeps SStables created by node operations 
+in a separate “maintenance” data set ignored by the regular compaction (STCS, LCS, etc.).
+
+After the node operation is completed, all the new SSTables kept in the maintenance set are 
+reshaped and then integrated into the main set of SStables to fall under the regular compaction strategy. 
+
+Off-strategy compaction prevents creating excessive compaction backlog due to a large number 
+of SStables generated by node operations. Compaction work during node operations is limited, 
+which speeds up the completion of ``repair``, ``bootstrap``, and others. 
+
+As a rule, we advise *against* aborting off-strategy compaction. However, if required, you 
+can abort the currently running off-strategy compaction with the following nodetool command: 
+
+.. code::
+   
+   nodetool stop RESHAPE 
+
+You can monitor off-strategy compaction via ScyllaDB logs, which contain information about when 
+off-strategy compaction is started and when it is completed.
```

Comment:
And: `nodetool compactionstats`; such compactions are listed under the RESHAPE type.